code/ui/ui_main.c

```diff
@@ -737,14 +759,22 @@ void _UI_Refresh( int realtime )
 		UI_BuildServerStatus(qfalse);
 		// refresh find player list
 		UI_BuildFindPlayerList(qfalse);
-	} 
-	
-	// draw cursor
-	UI_SetColor( NULL );
-	if (Menu_Count() > 0) {
-		UI_WideScreenMode(qtrue);
-		UI_DrawHandlePic(uiInfo.uiDC.cursorx * uiInfo.screenXFactorInv, uiInfo.uiDC.cursory, 48, 48, uiInfo.uiDC.Assets.cursor);
-		UI_WideScreenMode(qfalse);
+
+		// draw cursor
+		if ( (trap_Key_GetCatcher() & KEYCATCH_UI) && Menu_Count() > 0 ) {
+			UI_SetColor(NULL);
+			if ( (ui_widescreenCursorScale.integer & 2) && mvapi >= 3 && uiInfo.portraitMode && isMainMenu ) { //Scale height on the main menu in portrait mode.
```

Comment:
is portraitMode and mainMenu needed here? it only affects if and how UI_WidescreenMode() is called